src/ext/issuance.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * Functions for handling issue requests and server responses
+ *
+ * @author: Alex Davidson
+ */
+
+/* exported validateAndStoreTokens */
+/* exported BuildIssueRequest */
+
+const CACHED_COMMITMENTS_STRING = "cached-commitments";
+
+// Creates an issuance request for the current set of stored tokens. The format
+// is base64(json(BlindTokenRequest)) where BlindTokenRequest
+// corresponds to the following Go struct:
+//
+// type BlindTokenRequest struct {
+//      Type     ReqType  `json:"type"`
+//      Contents [][]byte `json:"contents"`
+// }
+//
+// Note that Go will automatically render and decode []byte as base64 encoded
```

Comment:
I am unsure myself actually, this comment is copy-pasted from the original file so it could be quite old and I'm not exactly sure of its origin. I think I'll rewrite it in the @param format and then remove this part to save confusion.